pages/data-providers/index.jsx

```diff
@@ -112,6 +112,9 @@ const DataProviderPage = ({ store }) => {
   }, [])
 
   const handleSubmitForm = () => {
+    // Form most likely submitted with enter in safari
+    if (!dataProvider.created) return
```

Comment:
If it's not boolean, it's better to compare it as an object.

```suggestion
    if (dataProvider.created == null) return
```